libs/solid/skeleton/src/lib/components/feedback/feedback.component.ts

```diff
@@ -9,23 +14,30 @@ import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
 })
 export class FeedbackComponent implements OnInit, OnDestroy {
   private static STORAGE_KEY = 'FEEDBACK';
-  public Form: FormGroup;
+  private title : string = "";
+  public Form: FormGroup; // "strictPropertyInitialization": false in tsconfig.base.json so that we don't have to initialize the value 
```

Comment:
Siehe Kommentar unten. Ich bin nicht so sicher, ob das eine gute Idee ist weil das evtl. auch Auswirkungen auf andere Stellen im Code hat.

Wir sollten nicht für eine spezifische Lösung die grundlegenden Parameter ändern.

Reply:
Probier mal, Zeile 18 in den `constructor `zu verschieben (unter die `FormBuilder`-Deklaration).
Zeile 17 müsste heißen
```
private title = '';
```
Da '' schon impliziert, dass es ein String ist, muss hier nicht nochmal der Typ deklariert werden. 
Eigentlich sollten es auch Single Quotes sein - das wird bei mir beim Speichern automatisch von Prettier geändert.
In der Datei `.prettierrc` (root Verzeichnis) steht `SingleQuote = true`.